src/main.ts

```diff
@@ -36,6 +31,68 @@ export async function createProject(options: Options) {
         task: () => copyTemplateFiles(options, templateDirectory, targetDirectory),
       },
       {
+        title: "📦 Installing dependencies with yarn, this could take a while",
+        task: async (_, task): Promise<void> => {
+          const execute = command("yarn install", { cwd: targetDirectory });
+
+          let outputBuffer: string = ""; // Buffer to store output characters
+
+          const maxChunks = 1; // Define the number of chunks to display
+          const chunkSize = 1024; // Define the size of each chunk (1KB here)
+
+          // Handle stdout
+          execute?.stdout?.on("data", (data: Buffer) => {
+            outputBuffer += data.toString(); // Append data to the buffer
+
+            // Ensure the buffer doesn't exceed the size of maxChunks * chunkSize
+            if (outputBuffer.length > maxChunks * chunkSize) {
+              outputBuffer = outputBuffer.slice(-maxChunks * chunkSize); // Keep only the last N chunks
+            }
+
+            // Add a forced newline if needed (this makes it friendly to Listr2)
+            const visibleOutput =
+              outputBuffer
+                .match(new RegExp(`.{1,${chunkSize}}`, "g")) // Split into chunks
+                ?.slice(-maxChunks) // Keep only the last N chunks
+                .map(chunk => chunk.trimEnd() + "\n") // Ensure each chunk ends with a newline
+                .join("") ?? outputBuffer;
+
+            task.output = visibleOutput; // Set the trimmed and formatted output
+          });
+
+          // Handle stderr similarly
+          execute?.stderr?.on("data", (data: Buffer) => {
+            outputBuffer += data.toString();
+
+            if (outputBuffer.length > maxChunks * chunkSize) {
+              outputBuffer = outputBuffer.slice(-maxChunks * chunkSize);
+            }
+
+            const visibleOutput =
+              outputBuffer
+                .match(new RegExp(`.{1,${chunkSize}}`, "g")) // Split into chunks
+                ?.slice(-maxChunks)
+                .map(chunk => chunk.trimEnd() + "\n") // Force a newline at the end of each chunk
+                .join("") ?? outputBuffer;
+
+            task.output = visibleOutput;
+          });
+
+          await execute;
+        },
+        skip: () => {
+          if (!options.install) {
+            return "Manually skipped, since `--skip-install` flag was passed";
+          }
+          return false;
+        },
+
+        rendererOptions: {
+          outputBar: 8, // Optional, adjust for your needs
```

Comment:
and need to remove that type of comments and I think most of the comments from this pr